train_prototype.py

```diff
@@ -116,6 +120,26 @@ def train(tasks, model, args, device):
                 iter_loss, acc))
             running_loss = 0.0
 
+        # evaluate in k shot fashion
+        if iterations % args.eval_every == 0:
+            test_model.load_state_dict(model.state_dict())
+            test_mean, test_std = k_shot_testing(test_model, episodes, test_task, device, num_test_batches=args.num_test_batches)
+            writer.add_scalar('TestTask/Acc', test_mean)
+            writer.add_scalar('TestTask/STD', test_std)
```

Comment:
Should we also pass the iterations as the third parameter? So we know after how many iterations we get a particular value. I'm unsure how does it work without it, does it take the current time?
`writer.add_scalar('TestTask/Acc', test_mean, iterations)`

Reply:
No you are right, it should be as you wrote above!